Services/Images/ImageService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Images;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+using System.IO;
+
+namespace Andculture.Sitefinity.Testing.Services.Images
+{
+    public class ImageService : ODataService<Image>
+    {
+        public ImageService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/images";
+        }
+
+        public IRestResponse Upload(Image model)
+        {
+            string documentTitle = "mobile-marketing";
+            string documentExtension = ".jpg";
+            string docFilePath = string.Concat(Directory.GetCurrentDirectory(), @"\Media\", documentTitle, documentExtension);
+            var jObject = JObject.FromObject(model);
+            jObject.Add("DirectUpload", true);
+
+            var url = Settings.BaseUrl + EndpointUrl;
```

Comment:
If the previous abstraction was made you'd be able to use it here as well. It would reduce the boilerplate code.